src/LeanCode.ContractsGenerator/Analyzers/InternalStructureCheck.cs

```diff
@@ -0,0 +1,56 @@
+namespace LeanCode.ContractsGenerator.Analyzers
+{
+    public class InternalStructureCheck : BaseAnalyzer
+    {
+        public const string Code = "LNCD001";
```

Comment:
Don't we already have such (or very similar) code in analyzers from our CoreLib?